### ISSUE TITLE
this didn't detect as a csv

### ISSUE THREAD

**Issue by karissa** [finnp/detect-csv#7]
Maybe a csv like this should pass? Just ran in to this with some real-world data while using dat

```
"name"
"finnp"
"karissa"
"mafintosh"
"albert"
```

**Comment by finnp** [comment]
I have just commited support for quoted strings. But I am thinking about also allowing other one-column csv like this
```
hello
1
2
3
4
```

However I am not exactly sure yet how to differentiate them from things like this
```
<xml>
what
</xml>
```

**Comment by karissa** [comment]
Yeah, good point. perhaps we could just close this and the consumer can
decide what to do if it returns null

On Thursday, August 6, 2015, Finn Pauls <notifications@github.com> wrote:

> I have just commited support for quoted strings. But I am thinking about
> also allowing other one-column csv like this
>
> hello
> 1
> 2
> 3
> 4
>
> However I am not exactly sure yet how to differentiate them from things
> like this
>
> <xml>
> what
> </xml>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/finnp/detect-csv/issues/7#issuecomment-128371552>.
>


-- 
Karissa McKelvey
http://karissa.github.io/ <http://karissamck.com>

Send me an encrypted message:
https://keybase.io/karissa <https://keybase.io/karissa/key.asc>
